src/coreclr/gc/gc.cpp

```diff
@@ -53336,6 +53465,7 @@ bool gc_heap::compute_memory_settings(bool is_initialization, uint32_t& nhp, uin
     }
 
     m_high_memory_load_th = min ((high_memory_load_th + 5), v_high_memory_load_th);
+    almost_high_memory_load_th = max((high_memory_load_th - 5), 1u);
 
```

Comment:
since this can be set by a config it can be smaller than 5 so this can't be an unsigned number.

Reply:
good catch, thanks - note also that ~25 lines up there was a possibility for overflow